### ISSUE TITLE
Default colours no longer show

### ISSUE THREAD

**Issue by maddisondesigns** [WordPress/gutenberg#7041]
**Describe the bug**
In my theme I've added support for wide and full images using `add_theme_support( 'align-wide' );`. I haven't specified any colours though.

When I click on a Paragraph block and go to select a Background or Text colour, I'm not given any default colours. It simply shows 4 empty colour selectors.

**To Reproduce**
1. Insert Paragraph Block
2. Click on Paragraph Block
3. Click on 'Block' tab in sidebar
4. Click on 'Background Color' or 'Text Color' and try to select a default colour

**Expected behavior**
Gutenberg used to provide a default selection of colours. If this is no longer the case, then those blank colour selectors should be removed as they serve no purpose.

**Screenshots**
![gutenberg_nodefaultcolours](https://user-images.githubusercontent.com/2078245/40769923-9404bfac-64fc-11e8-9c53-6ad4473a2df9.png)

_macOS 10.12.1 (16B2659)
Firefox Quantum 60.0.1 (64-bit)
Gutenberg 2.9.2_

**Comment by nfmohit-wpmudev** [comment]
I'm not able to reproduce the issue on my end, the default colors show up as expected for me:

![issue-7041](https://user-images.githubusercontent.com/20284937/40795120-ee90a2a8-6522-11e8-8600-b058386dd394.png)

_macOS 10.13.4
Firefox Quantum 60.0.1 (64-bit)
Gutenberg 2.9.2_

**Comment by maddisondesigns** [comment]
I've deactivated every plugin (except for Gutenberg) and switched the theme to the Gutenberg theme, and I still see a similar thing. I say similar because the blank colour selectors are now just block dots rather than black rings.

![gutenberg_nodefaultcolours2](https://user-images.githubusercontent.com/2078245/40817308-11d7bf86-6595-11e8-9a0f-17f1dc04ec58.png)

There doesn't appear to be any errors when I look at the console (just a warning about using the `gutenberg` array being deprecated, in the Gutenberg theme)

If I switch to TwentySeventeen, which doesn't add support for Gutenberg, then the default colours show.

Based on this very quick test, I would guess that the colours are being messed up when 'add_theme_support` is being used.

**Comment by Netzberufler** [comment]
The implementation for the editor color palette feature changed recently and now works with CSS classes and named color arrays.
https://github.com/WordPress/gutenberg/issues/6585

Make sure you don't have the old implementation anywhere in your theme, and also add the new custom color classes to your theme's stylesheet.

**Comment by danielbachhuber** [comment]
I'm unable to reproduce this with Twenty Seventeen active.

> I've deactivated every plugin (except for Gutenberg) and switched the theme to the Gutenberg theme, and I still see a similar thing.

Which "Gutenberg theme" are you referring to? Maybe this is what needs to be updated?

**Comment by maddisondesigns** [comment]
> Which "Gutenberg theme" are you referring to? Maybe this is what needs to be updated?

This was the Gutenberg theme I was referring to:
https://github.com/WordPress/gutenberg-starter-theme

I know this theme is quite out of date now but I tried it simply because it's got very minimal styling.

I originally encountered the issue when using a theme I developed myself, which is up to date with Gutenberg changes.

I've just updated to Gutenberg 3.0.0 and the default colours are showing again now, when using my own them and the Gutenberg theme.
